schema/home/fetch.js

```diff
@@ -22,6 +23,12 @@ export const featuredFair = () => {
   });
 };
 
+export const activeSaleArtworks = (accessToken) => {
+  return gravity.with(accessToken)('me/lot_standings').then((results) => {
```

Comment:
I omitted that to match the my-bids module experience, since we want to show all bids regardless of sale type. @devangt can you chime in? Do we want to show only live sales on eigen's home screen or timed ones as well.